### ISSUE TITLE
Any plan to make workflow to support kotlin multiplatform?

### ISSUE THREAD

**Issue by fengdai** [square/workflow#319]

**Comment by zach-klippenstein** [comment]
Yes, we do eventually want to do that. @rjrjr actually started the work as a hackweek project a couple weeks ago, but I'm not sure where it ended up. We will probably do this after v1.0.0.

**Comment by fengdai** [comment]
> Yes, we do eventually want to do that. @rjrjr actually started the work as a hackweek project a couple weeks ago, but I'm not sure where it ended up. We will probably do this after v1.0.0.

Fantastic!

**Comment by rjrjr** [comment]
It didn't get too far since our public API is so dependent on Okio. But the good news is that they're hard at work on mp themselves. With that done, I don't see much blocking us.

**Comment by akfpnd** [comment]
Any update so far? I willing to try how it works with KMP.

**Comment by rjrjr** [comment]
@akfpnd there's some work in progress, as a matter of fact. https://github.com/square/workflow-kotlin/pull/746
